### PR TITLE
Fixes for standalone tests

### PR DESCRIPTION
**Pull request by dzafman**

### CODE REVIEW

**Comment by dzafman** [comment]
Testing passed:
scrub.yaml (qa/standalone/scrub)
http://pulpito.ceph.com/dzafman-2018-06-09_13:44:30-rados-wip-zafman-testing-distro-basic-smithi/2646880
osd.yaml (qa/standalone/osd)
http://pulpito.ceph.com/dzafman-2018-06-09_13:43:44-rados-wip-zafman-testing-distro-basic-smithi/2646878

../qa/run-standalone.sh osd-backfill-stats.sh osd-recovery-stats.sh repro_long_log.sh osd-unexpected-clone.sh
PASSED

**Comment by dzafman** [comment]
retest this please

File: qa/standalone/scrub/osd-unexpected-clone.sh [hunk 1]
```diff
@@ -56,19 +56,19 @@ function TEST_recover_unexpected() {
     JSON=`objectstore_tool $dir $osd --op list foo | grep snapid.:1`
     echo "JSON is $JSON"
     rm -f $dir/_ $dir/data
-    objectstore_tool $dir $osd "$JSON" get-attr _ > $dir/_
-    objectstore_tool $dir $osd "$JSON" get-bytes $dir/data
+    objectstore_tool $dir $osd "$JSON" get-attr _ > $dir/_ || return 1
```
Comment (tchaikov) [inline qa/standalone/scrub/osd-unexpected-clone.sh]: nit, could add
```sh
set -e
```

at the top of the script

File: qa/run-standalone.sh [hunk 2]
```diff
@@ -94,6 +94,15 @@ if [ "$precore" = "$COREPATTERN" ]; then
...
     sudo sysctl -w ${KERNCORE}=${COREPATTERN}
 fi
+# Clean out any cores in core target directory (currently .)
+if ls $(dirname $(sysctl -n $KERNCORE)) | grep -q '^core\|core$' ; then
+    mkdir found.cores.$$ 2> /dev/null || true
+    for i in $(ls $(dirname $(sysctl -n $KERNCORE)) | grep '^core\|core$'); do
+	mv $i found.cores.$$
+    done
+fi
+echo "Stray cores put in $(pwd)/found.cores.$$"
```
Comment (tchaikov) [inline qa/run-standalone.sh]: might want to move this line into the `if` block ?

**Pull request merged**